src/genTypeFlow.ts

```diff
@@ -136,6 +136,7 @@ export function getLoaderType(resourceConfig: ResourceConfig, resourcePath: Read
 }
 
 export function getLoadersTypeMap(
+    language: LanguageOptions = LanguageOptions.FLOW,
```

Comment:
fwiw, this file is called `genTypeFlow.ts` implying this is specific for generating types for flow. in my mind, for doing typescript type generation, we'd just copy and paste this whole file and start again making it only typescript focused (rather than having each individual function do a little if statement to check are we generating for flow or typescript)

with that in mind, the choice of 'flow' here would be implicit and not needed as an argument